1.0.0-beta7/README.md

```diff
@@ -14,6 +14,7 @@ This project is part of ASP.NET 5. You can find samples, documentation, and gett
 ## Supported tags
 
 * [`1.0.0-beta7`, `latest`  _(1.0.0-beta7/Dockerfile)_](https://github.com/aspnet/aspnet-docker/blob/master/1.0.0-beta7/Dockerfile)
+* [`1.0.0-beta7`, `coreclr`  _(1.0.0-beta7/Dockerfile)_](https://github.com/aspnet/aspnet-docker/blob/master/1.0.0-beta7-coreclr/Dockerfile)
```

Comment:
@jchannon this line should just be `coreclr-1.0.0-beta7`, I don't think we need to maintain a `coreclr` tag for now.